pkg/kubefed2/federate/util.go

```diff
@@ -25,6 +25,7 @@ import (
 var systemMetadataFields = []string{"selfLink", "uid", "resourceVersion", "generation", "creationTimestamp", "deletionTimestamp", "deletionGracePeriodSeconds"}
 
 func RemoveUnwantedFields(resource *unstructured.Unstructured) {
+	unstructured.RemoveNestedField(resource.Object, "status")
```

Comment:
This looks like a bug discovered during implementation of yaml output, and accordingly seems deserving of its own commit.

Reply:
Put into another commit!